filer/models/filemodels.py

```diff
@@ -506,16 +511,43 @@ def __lt__(self, other):
 
     @property
     def actual_name(self):
+        clean_name = self.clean_actual_name
+        if not self.sha1:
+            try:
+                self.generate_sha1()
+            except Exception:
```

Comment:
I was no confident in what exceptions could be raised here since file operations and hashlib is used.
This handling was here for a long time https://github.com/pbs/django-filer/blob/6c1b462eb48d4aa5add4a670b0edfb65d2322807/filer/models/filemodels.py#L295 and missing some exception type would be worse.